src/amo/sagas/collections.js

```diff
@@ -244,6 +249,7 @@ export function* modifyCollection(
     }
   } catch (error) {
     log.warn(`Failed to ${type}: ${error}`);
+    yield put(finishCollectionModification());
```

Comment:
Given this is run in either scenario (`try`/success and `catch` failure), wouldn't it make more sense to put `yield put(finishCollectionModification());` in a `finally {}` block? It changes the order, but I don't think that should matter.

Reply:
Great idea, thanks!